src/components/bubble/_bubble.scss

```diff
@@ -8,48 +8,54 @@ $borderSize: 3px;
 $triangleSize: 12px;
 $innerTriangleSize: $triangleSize - 2 * $borderSize + 1;
 
-.mint-bubble {
-  @include component;
-  overflow: visible;
-  background: $bubbleBackground;
-  border: $borderSize solid $bubbleBorderColor;
-  border-radius: $bubbleBorderRadius;
-  padding: 15px;
-  box-shadow: $bubbleBoxShadow;
-
-  &:before {
-    content: "";
-    border: $triangleSize solid transparent;
-    border-bottom-color: $bubbleBorderColor;
-    position: absolute;
-    top: -(2 * $triangleSize);
-    right: 0;
-    left: 0;
-    width: 0;
-    margin: 0 auto;
-  }
+$include-html-classes: true !default;
```

Comment:
So, after discussion with @matzimowski @kdzwinel we've decided that this variable should be `false` by default. It will be set to true in `main.scss` in styleguide.

In other cases it will result in a very handy `import`, where you can be sure that you won't get classes from styleguide baked-in in your component.

Implementation details up to @matzimowski